meta-mender-imx/README.md

```diff
@@ -0,0 +1,58 @@
+# meta-mender-imx
+
+Mender integration for imx based boards
+
+The supported and tested boards are:
+
+ - [i.MX8M Nano EVK](TBD)
+
+Visit the individual board links above for more information on status of the
+integration and more detailed instructions on how to build and use images
+together with Mender for the mentioned boards.
+
+## Dependencies
+
+This layer depends on:
+
+```
+URI: https://github.com/Freescale/meta-freescale-3rdparty
+branch: zeus
+revision: HEAD
+```
+
+```
+URI: https://github.com/Freescale/meta-freescale-distro
+branch: zeus
+revision: HEAD
+```
+
+```
+URI: https://github.com/Freescale/meta-freescale
+branch: zeus
+revision: HEAD
+```
```

Comment:
As this is based on the NXP BSP, there is a lot more to it here.

I would just say,

```
This layer depends on NXP BSP `imx-5.4.3-2.0.0`
```

Reply:
Fixed